examples/nlp/gpt/conf/training_rm.yaml

```diff
@@ -70,6 +70,10 @@ model:
   global_batch_size: 64
   megatron_amp_O2: True
 
+  # if the inference microbatch size is big, it's possible
+  # to split it using forward mbs and run inference iteratively
+  forward_micro_batch_size: ${.micro_batch_size}
```

Comment:
ebb69f44523a7712707effe9589365a69c66c746

Reply:
I was also expecting it to be used as mbs for validation, isn't it the case?

Reply:
for validation in the reward model we call [`self.model.get_loss_and_metrics(batch=batch, forward_only=True)
`](https://github.com/NVIDIA/NeMo-Aligner/blob/main/nemo_aligner/algorithms/supervised.py#L92). This is really so we can get all the training metrics but on validation, instead of getting just the reward so this path is not hit during training

Reply:
Ok, it's a bit confusing that `forward_only=True` doesn't actually use `forward_micro_batch_size` => I wonder if we should have named this option `inference_micro_batch_size` (same in the PPO actor by the way).
But it's a bit late now => not a huge deal, resolving.